types/src/front/lib/assistant.ts

```diff
@@ -79,6 +79,18 @@ export type ModelConfigurationType = {
   shortDescription: string;
   supportsMultiActions: boolean;
   isLegacy: boolean;
+
+  // Allows configuring parsing of special tags in the streamed model output.
+  tagsConfiguration?: {
```

Comment:
let's call it "delimiters" instead of "tags" since the system is applicable to more than "tags"?

Reply:
Fair 👍